cpp/data/installer/scripts/install_ubuntu_packages.sh

```diff
@@ -33,7 +33,7 @@ apt-get --quiet --yes --allow-unauthenticated install \
     mysql-utilities poppler-utils unzip uuid-dev tesseract-ocr tesseract-ocr-bul tesseract-ocr-ces tesseract-ocr-dan tesseract-ocr-deu \
     tesseract-ocr-eng tesseract-ocr-fin tesseract-ocr-fra tesseract-ocr-heb tesseract-ocr-hun tesseract-ocr-ita tesseract-ocr-lat \
     tesseract-ocr-nld tesseract-ocr-nor tesseract-ocr-pol tesseract-ocr-por tesseract-ocr-rus tesseract-ocr-script-grek tesseract-ocr-slv \
-    tesseract-ocr-spa tesseract-ocr-swe libdb-dev tcl-expect-dev tidy incron
```

Comment:
please restore alphabetical order
- libdb-dev should be after libcurl4-gnutls-dev
- incron should be after imagemagick
- sqlite3 should be after poppler-utils
- (unzip & uuid-dev should be after tcl-expect-dev)